### PR TITLE
Grim Syndicate missing info and repo

### PR DESCRIPTION
**Pull request by grimAgent**

### CODE REVIEW

**Review by eherrerosj — CHANGES_REQUESTED** [review]
You need to create a new ecosystem for "Grim Syndicate". For that, you need to create a new toml file like grim-syndicate.toml with title "Grim Syndicate" and add the org in it (https://github.com/Grim-Syndicate)

**Comment by grimAgent** [comment]
> You need to create a new ecosystem for "Grim Syndicate". For that, you need to create a new toml file like grim-syndicate.toml with title "Grim Syndicate" and add the org in it (https://github.com/Grim-Syndicate)

Thanks, just added it. Let me know if anything else is needed

File: data/ecosystems/s/solana.toml [hunk 1]
```diff
@@ -321,6 +322,7 @@ github_organizations = [
   "https://github.com/firedancer-io",
   "https://github.com/generatus",
   "https://github.com/GenesysGo",
+  "https://github.com/Grim-Syndicate",
```
Comment (eherrerosj) [inline data/ecosystems/s/solana.toml:325]: this line can be removed, out ETL takes care of rolling all up

File: data/ecosystems/s/solana.toml [hunk 2]
```diff
@@ -48254,6 +48256,9 @@ url = "https://github.com/Grim-R3ap3r/Solana-Project"
 [[repo]]
 url = "https://github.com/Grim-Syndicate/eta-backend"
 
+[[repo]]
+url = "https://github.com/Grim-Syndicate/eta-frontend"
```
Comment (eherrerosj) [inline data/ecosystems/s/solana.toml:48260]: this line can be removed, out ETL takes care of rolling all up

**Review by eherrerosj — CHANGES_REQUESTED** [review]
small last changes

File: data/ecosystems/s/solana.toml (hunk 1 above)
Comment (grimAgent) [reply data/ecosystems/s/solana.toml]: done

File: data/ecosystems/s/solana.toml (hunk 2 above)
Comment (grimAgent) [reply data/ecosystems/s/solana.toml]: and done

**Review by eherrerosj — APPROVED** [review]
great thx!

**Pull request merged**